media-gfx/openexr_viewers/openexr_viewers-2.2.1.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools flag-o-matic
+
+DESCRIPTION="OpenEXR Viewers"
+HOMEPAGE="http://openexr.com/"
+SRC_URI="http://download.savannah.gnu.org/releases/openexr/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/23" # based on SONAME
+KEYWORDS="~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+IUSE="cg opengl"
+
+RDEPEND="~media-libs/ilmbase-${PV}:=
+	~media-libs/openexr-${PV}:=
+	>=media-libs/ctl-1.5.2:=
+	x11-libs/fltk:1[opengl]
+	opengl? (
+		virtual/opengl
+		x11-libs/fltk:1[opengl]
+		cg? ( media-gfx/nvidia-cg-toolkit )
+	)"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+RESTRICT="test"
+
+PATCHES=( "${FILESDIR}/${PN}-2.2.0-Remove-nVidia-automagic.patch" )
+
+src_prepare() {
+	default
+	sed -i -e 's:AM_CONFIG_HEADER:AC_CONFIG_HEADERS:' configure.ac || die
+	eautoreconf
+}
+
+src_configure() {
+	local myconf
+
+	if use cg; then
+		myconf="--with-cg-prefix=/opt/nvidia-cg-toolkit"
+		append-flags "$(no-as-needed)" # binary-only libCg is not properly linked
+	fi
+
+	econf \
+		$(use_enable cg) \
+		$(use_with opengl fltk-config /usr/bin/fltk-config) \
+		${myconf}
+}
+
+src_install() {
+	emake \
+		DESTDIR="${D}" \
+		docdir=/usr/share/doc/${PF}/pdf \
```

Comment:
seeing that you need to eautoreconf anyways, could you look into fixing this too? It likely only needs removing of `docdir` in some `Makefile.am`

Reply:
I find that I'm overriding some form of the doc directory variable in most programs I maintain.

Reply:
Are you doing that to make stuff install to `pdf/` subdirectory? If that's just 1-2 files, it's completely unnecessary.

Reply:
@mgorny this is the way it was in the original ebuild. I think it is only one or two files. I need to double check.